pkg/apis/build/v1alpha1/build_source.go

```diff
@@ -11,9 +11,12 @@ import (
 // BuildSourceType enumerates build source type names.
 type BuildSourceType string
 
-// LocalCopy defines a source type that waits for user input, as in a local copy into a POD.
+// LocalCopy source type when the user will upload local content.
```

Comment:
not officially asking for a change, but am wondering .... is it truly obvious that "uploading local content" means it is *NOT* a remote artifact?

Probably the answer is "yes it is obvious", but at least at this precise moment, I have a small percent of uncertainty

But especially when we start adding remote artifact types other than HTTP, perhaps distinguishing that local source is not one of the remote artifacts becomes a bit more important.

I won't push on it further @otaviof but if you are amenable to making it *really* clear by all means

Reply:
the comments in question are clearer @otaviof 

after reading your comment here and in the code, I think I might even go a step further, and 
- change constants starting with RemoteArtifacts... to DownloadedArtfacts...
- change LocalCopy to UploadedArtifacts

WDYT?